src/list.json

```diff
@@ -68,6 +68,14 @@
       }
     ],
     "D": [
+      {
```

Comment:
Please Re-style the code via pasting this command in the Repo Directory
`git checkout <your branch>`
`curl https://restyled.io/gh/monizb/repos/hacktoberfest-swag-list/jobs/2338748/patch | git am`
`git push`